test/e2e/README.md

```diff
@@ -89,34 +58,73 @@ Describe: <Role>
        Expect: <Result> (detailed checks for execution results)
 ```
 
-### 5. Adding New Test Data
+### 8. Adding New Test Data
+
+#### 8.1 Command Suite
+Command suite uses pre-baked test data, which is a bunch of layered archive files compressed from registry storage. Test data are all stored in `$REPO_ROOT/test/e2e/testdata/distribution/` but separated in different sub-folders: oras distribution uses `mount` and upstream distribution uses `mount_fallback`.
```

Comment:
What's the difference between `mount` and `mount_fallback`?

Reply:
`mount` is used by oras distribution and it can contains artifact manifests and referrers.

`mount_fallback` is used by upstream distribution and it only contains image manifests and referrer tags.